cli/permissions.rs

```diff
@@ -170,51 +180,108 @@ impl DenoPermissions {
   pub fn check_read(&self, filename: &str) -> DenoResult<()> {
     match self.allow_read.get_state() {
       PermissionAccessorState::Allow => Ok(()),
-      PermissionAccessorState::Ask => match self
-        .try_permissions_prompt(&format!("read access to \"{}\"", filename))
-      {
-        Err(e) => Err(e),
-        Ok(v) => {
-          self.allow_read.update_with_prompt_result(&v);
-          v.check()?;
+      state => {
+        if check_path_white_list(filename, &self.read_whitelist) {
           Ok(())
+        } else {
+          match state {
+            // This shouldn't be possible but I guess rust doesn't realize.
```

Comment:
I'm not sure what this comment means?

Reply:
Should I remove the type as well, and just replace with a `_ => unreachable!()`?

Reply:
Or keep the type. I just think the comment is confusing.